test/videojs-contrib-hls.test.js

```diff
@@ -2769,6 +2775,13 @@ QUnit.test('blacklists playlist if key requests fail', function(assert) {
   hls.mediaSource.trigger('sourceopen');
   this.requests.shift()
     .respond(200, null,
+              '#EXTM3U\n' +
```

Comment:
Why is this now responding with a master playlist? The URL above (provided as `src`) seems to indicate that it is a media playlist.